src/app/action-items/action-items.component.ts

```diff
@@ -14,38 +15,41 @@ export interface ActionItemMapped extends ActionItem {
   templateUrl: './action-items.component.html',
   styleUrls: ['./action-items.component.css']
 })
-export class ActionItemsComponent implements OnInit {
+export class ActionItemsComponent implements OnInit, OnDestroy {
   dataSource: ActionItemMapped[];
-  isloadingActionItems = false;
-  private currentDate: Date = new Date();
+  isLoadingActionItems = false;
+  isLoadingActionItem = false;
+  actionItemId: string | undefined;
+  actionItem: ActionItem;
+  errorMessage: string | undefined;
+  justAddedItem: ActionItem;
```

Comment:
**FIX:** Try to prepare solution without this variable. Hint: you have all the information that you need after creation, so try to start from the code responsible for that.

Reply:
@romatowskak this is still not removed. I think that it can be safely removed as you're only assigning it and not doing anything with it after.